doc/UsersGuide/source/SSP.rst

```diff
@@ -0,0 +1,176 @@
+
+.. code-block:: xml
+  :emphasize-lines: 10-19,27-36,47-56
+
+  <?xml version="1.0" encoding="UTF-8"?>
+  <ssd:SystemStructureDescription name="Test" version="Draft20180219">
+    <ssd:System name="Root">
+      <ssd:Elements>
+        <ssd:System name="WC2">
+          <ssd:Connectors>
+            <ssd:Connector name="C1" kind="input" type="Real"/>
+            <ssd:Connector name="C2" kind="output" type="Real"/>
+          </ssd:Connectors>
+          <ssd:Annotations>
+            <ssc:Annotation type="org.openmodelica">
+              <oms:Bus name="bus2">
+                <oms:Signals>
+                  <oms:Signal name="C1"/>
+                  <oms:Signal name="C2"/>
+                </oms:Signals>
+              </oms:Bus>
+            </ssc:Annotation>
+          </ssd:Annotations>
+        </ssd:System>
+        <ssd:System name="WC1">
+          <ssd:Connectors>
+            <ssd:Connector name="C1" kind="output" type="Real"/>
+            <ssd:Connector name="C2" kind="input" type="Real"/>
+            <ssd:Connector name="C3" kind="input" type="Real"/>
+          </ssd:Connectors>
+          <ssd:Annotations>
+            <ssc:Annotation type="org.openmodelica">
+              <oms:Bus name="bus1">
+                <oms:Signals>
+                  <oms:Signal name="C1"/>
+                  <oms:Signal name="C2"/>
+                </oms:Signals>
+              </oms:Bus>
+            </ssc:Annotation>
+          </ssd:Annotations>
+        </ssd:System>
+      </ssd:Elements>
+      <ssd:Connections>
+        <ssd:Connection startElement="WC2" startConnector="C1"
+                        endElement="WC1" endConnector="C1"/>
+        <ssd:Connection startElement="WC2" startConnector="C2"
+                        endElement="WC1" endConnector="C2"/>
+        <ssd:Connection startElement="WC2" startConnector="C2"
+                        endElement="WC1" endConnector="C3"/>
+      </ssd:Connections>
+      <ssd:Annotations>
+        <ssc:Annotation type="org.openmodelica">
+          <oms:Connections>
+            <oms:Connection startElement="WC1" startConnector="bus1"
+                            endElement="WC2" endConnector="bus2"/>
+            <oms:Connection startElement="WC2" startConnector="bus2"
+                            endElement="WC1" endConnector="C3"/>
+          </oms:Connections>
+        </ssc:Annotation>
+      </ssd:Annotations>
+    </ssd:System>
+  </ssd:SystemStructureDescription>
+
+.. index:: SSP; TLM Systems
+
+TLM Systems
+###########
+
+TLM systems are only allowed on top-level. SSD annotations are used to specify the system type inside the ``ssd:SimulationInformation`` tag, as shown in the example below.
+
+.. code-block:: xml
+  :emphasize-lines: 4-8
+
+  <?xml version="1.0"?>
+  <ssd:System name="tlm">
+    <ssd:SimulationInformation>
+     <ssd:Annotations>
+        <ssd:Annotation type="org.openmodelica">
+          <oms:tlmMaster />
```

Comment:
I think we should mention the available attributes, e.g. ip and port.

Reply:
I think `oms:tlmMaster` should be `oms:TlmMaster` to be consistent with the rest of the xml. See #394

Reply:
Ok, I added attribute description and renamed the tag.